src/pages/create/Create.test.js

```diff
@@ -28,12 +28,15 @@ describe('Create', () => {
 
     userEvent.click(getByText('Create incident', { selector: 'button' }))
     expect(createIncident).toHaveBeenCalledTimes(1)
-    expect(createIncident).toHaveBeenCalledWith({
-      description: 'Dancing Elvis',
-      services: ['flights'],
-      title: 'Whoo hoo!',
-      type: 'degraded',
-    })
+    expect(createIncident).toHaveBeenCalledWith(
+      { email: '' },
```

Comment:
To avoid having an empty email you can create a context inside the tests: https://www.polvara.me/posts/mocking-context-with-react-testing-library/

Reply:
Done.